src/Avalonia.Labs.RoutedCommand/RoutedCommandsManager.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Avalonia.Input;
+using Avalonia.Interactivity;
+
+namespace Avalonia.Labs.Input;
+
+public sealed class RoutedCommandsManager : AvaloniaObject
+{
+    public static RoutedEvent<CanExecuteRoutedEventArgs> CanExecuteEvent { get; } =
+        RoutedEvent.Register<CanExecuteRoutedEventArgs>("CanExecute", RoutingStrategies.Bubble, typeof(RoutedCommandsManager));
+
+    public static RoutedEvent<ExecutedRoutedEventArgs> ExecutedEvent { get; } =
+        RoutedEvent.Register<ExecutedRoutedEventArgs>("Executed", RoutingStrategies.Bubble, typeof(RoutedCommandsManager));
+
+    public static readonly AttachedProperty<IList<RoutedCommandBinding>> CommandsProperty =
+        AvaloniaProperty.RegisterAttached<RoutedCommandsManager, InputElement, IList<RoutedCommandBinding>>("Commands");
+
+    private static readonly WeakReference<IInputElement?> _inputElement = 
+        new WeakReference<IInputElement?>(default);
+
+    internal static event EventHandler? Invalidate = default;
```

Comment:
Shouldn't it be public and named RequerySuggested? 
Similarly to https://github.com/dotnet/wpf/blob/main/src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Input/Command/CommandManager.cs#L38